mxcube3/routes/Queue.py

```diff
@@ -939,13 +939,15 @@ def serialize_queue_to_json():
             if child['py/object'].split('.')[1] == 'TaskGroup': #keep going down
                 for grandChild in child['_children']:
                     if grandChild['py/object'].split('.')[1] == 'TaskGroup': #keep going down one more time for the Char
+                        """ the following loop is not needed
```

Comment:
Okey, I was afraid that the queue_to_json in qutils did not work as you wanted. So please if you are not using this method remove it.